packages/amqp/package.json

```diff
@@ -25,7 +25,7 @@
     "prepublishOnly": "npm run build:release"
   },
   "dependencies": {
-    "@lokalise/node-core": "^13.0.1",
+    "@lokalise/node-core": "^13.0.0",
```

Comment:
this is not a peer dependency, so I would use the latest available version here, to force the bump

Reply:
Correct me if I am wrong, but this will allow us to be compatible with `13.x.x`, right? using `^13.1.0` it will be `13.1.x`.
I have no strong opinion but I thought that leaving it open to minor updates would be good. If you disagree I will use your suggestion

Reply:
npm always aim to minimize amount of versions in use. If it can establish a single version that will satisfy constraints from different users, it will use that - e. g. if you have "peerDependency > 12" from package A, "dependency ^13.0.0" from package B and "dependency ^13.1.0" from package C, it will resolve version to 13.1.0, as it satisfies all three constraints.

in this case 13.1.0 is the version where we fixed backwards compatibility (hiddent the symbol that was breaking interface previously), so I think that enforcing it should actually reduce amount of compatibility problems that we will have

Reply:
Thanks for the explanation Igor! it makes a lot of sense, will apply your suggestion and release 🙏